src/components/CustomTextInput/CustomTextInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { Text, TextInput, View, Dimensions } from 'react-native';
+
+import { useState } from 'react';
+
+import { colors } from '../../constants/colors';
+
+import IconButton from '../IconButton/IconButton';
+
+import { ICustomTextInputProps } from './customTextInput.types';
+
+import styles from './customTextInput.styles';
+
+const iconSize = Dimensions.get('window').height * 0.025;
+
+const CustomTextInput = ({
+  title,
+  hintText,
+  secureTextEntry,
+  value,
+  setValue,
+  style,
+}: ICustomTextInputProps) => {
+  const [focused, setFocused] = useState(false);
+  return (
+    <View style={[styles.container, style]}>
+      <Text style={styles.title}>{title}</Text>
+      <View style={[styles.textInputContainer, focused && styles.focused]}>
+        {hintText !== null && <Text style={styles.hintText}>{hintText}</Text>}
```

Comment:
use shorthandle {!!hintText && <Text style={styles.hintText}>{hintText}</Text>} 
[docs](https://www.codingem.com/javascript-double-exclamation-operator/#:~:text=In%20JavaScript%2C%20the%20double%20exclamation,%E2%80%9Ctruthy%E2%80%9D%20objects%20become%20true)